src/expandableCalendar/index.js

```diff
@@ -448,7 +473,13 @@ class ExpandableCalendar extends Component {
 
   renderKnob() {
     // TODO: turn to TouchableOpacity with onPress that closes it
-    return (
+    return this.props.disablePan ? (
+      <TouchableOpacity style={this.style.knobContainer} onPress={() => this.onKnobPress()}>
+        <View style={this.style.knobContainer} pointerEvents={'none'}>
+          <View style={this.style.knob} testID={CALENDAR_KNOB}/>
+        </View>
+      </TouchableOpacity>
```

Comment:
When pressing the knob a subtle effect happens:
![Untitled](https://user-images.githubusercontent.com/12599839/81944160-e29cc700-95eb-11ea-970c-00730776c01c.png)
Moving `TouchableOpacity` inside `View` fixed it for me.

Also, I think we can make use of `leftArrowImageSource` and `rightArrowImageSource` with a `90deg` rotation to make it obvious that it's pressable.

```suggestion
      <View style={this.style.knobContainer}>
        <TouchableOpacity
          activeOpacity={0.75}
          style={this.style.knobContainer}
          onPress={() => this.onKnobPress()}>
          <Image
            source={
              isOpen
                ? this.props.leftArrowImageSource
                : this.props.rightArrowImageSource
            }
            style={[this.style.arrowImage, {transform: [{ rotate: '90deg' }]}]}
            testID={CALENDAR_KNOB}
          />
        </TouchableOpacity>
      </View>
```